docs/src/pages.ts

```diff
@@ -4,7 +4,12 @@ export interface MuiPage {
   pathname: string;
   children?: MuiPage[];
   disableDrawer?: boolean;
-  displayNav?: boolean;
+  /**
+   * Pages are considered to be ordered depth-first.
+   * If a page should be excluded from this order, set `order: false`.
+   * You want to set `ordered: false` if you don't want the page to appear in an ordered list e.g. for previous/next page navigation.
+   */
+  ordered?: boolean;
```

Comment:
The wording is a bit rough. `displayNav` also works considering the `disableDrawer` wording. Maybe just keep the original word but keep the description?